app/build.gradle

```diff
@@ -162,14 +162,18 @@ dependencies {
     implementation files('./lib/spotify-app-remote-release-0.7.2.aar')
 
     //Test Setup
+    def mockito_version = '2.7.1'       // Useful for mocking context
```

Comment:
We don't need Mockito, as I mentioned earlier.

Reply:
We already have our MockYimRepo

Reply:
How do i mock app context or any context for say?

Reply:
Done